### DISCUSSION TITLE
How to add a link to the copyright statement?

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [squidfunk/mkdocs-material#3263] [Q&A] [answered]
Is it possible to add a link to the copyright statement in `mkdocs.yml` within the `copyright` parameter, or do you need to [override](https://squidfunk.github.io/mkdocs-material/customization/#overriding-partials) the `footer` partial?

**Comment by squidfunk** [comment, marked as answer]
You can override the `footer.html` partial, but you should also be able to use inline HTML, i.e. you can define the link directly in `mkdocs.yml`. For more advanced customizations, [8.0.0](https://github.com/squidfunk/mkdocs-material/issues/3219) finally moves copyright into a new partial called `copyright`, which can be overridden without touching the footer. I hope to make the final release today.

**Reply by squidfunk** [reply]
I've added a new section to the docs how to [customize the copyright](https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-footer/#custom-copyright). 8.0.0 was released today, as well as Insiders 4.0.0 which both include the same changes 😊
